### PR TITLE
feature/input

### PR DESCRIPTION
**Pull request by BNS-MarkUlrich**

### CODE REVIEW

File: Assets/Scripts/Player/Interactor.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+using Minoord.Input;
+using UnityEngine.InputSystem;
+using System;
+
+namespace Examen.Player
+{
+    [RequireComponent(typeof(Pointer))]
```
Comment (swzwij) [inline Assets/Scripts/Player/Interactor.cs:7]: Is it needed for the interactor class to be depended on the pointer? shouldn't it be the other way around?

File: Assets/Scripts/Player/Interactor.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        private void OnDisable()
+        {
+            _interactAction.Disable();
+            _interactAction.performed -= OnInteractPerformed;
+        }
+
+        private void Awake() => _pointer = GetComponent<Pointer>();
+
+        private void OnInteractPerformed(InputAction.CallbackContext context)
+        {
```
Comment (swzwij) [inline Assets/Scripts/Player/Interactor.cs]: Pull from develop and do a IsOwner check so only the owner of this player can do this interaction

**Review by swzwij — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Player/Interactor.cs (hunk 1 above)
Comment (BNS-MarkUlrich) [reply Assets/Scripts/Player/Interactor.cs:7]: Would you point at what you interact with or interact with what you point at?

Comment (BNS-MarkUlrich) [reply Assets/Scripts/Player/Interactor.cs:7]: I would say the interactor is dependant on the pointer, as you can't interact with something without "clicking" (pointing) at it first. I've slightly refactored the code to better reflect this. Let me know what you think.

**Review by swzwij — APPROVED** [review]

**Pull request merged**